### ISSUE TITLE
Allow user to change default value of shouldEndSession

### ISSUE THREAD

**Issue by Glogo** [Accenture/alexia#32]
Add a way to change default value of `shouldEndSession`. Currently its set to `true`. This is OK for simple command-based apps.

For more conversational apps it is more suitable to have `shouldEndSession` set to `false`.

**How can user configure it (discuss and choose one):**
a) Provide the information in app config:
```
// Choose shorter name for config option
const app = alexia.createApp('appName', {shouldEndSessionByDefault: false});
```
b) Or specify the value after creating the app:
```
// The same, decide on shorter name
app.setShouldEndSessionByDefault(false);
```

**Comment by xtools-at** [comment]
I think b) is nicer, but that's just how I feel about it

**Comment by Glogo** [comment]
@xtools-at yes I agree, but why not both? :)
thx @misyak - resolved on #35
